app/javascript/components/enrollment/steps/Contact.js

```diff
@@ -14,12 +14,12 @@ const phoneUtil = libphonenumber.PhoneNumberUtil.getInstance();
 class Contact extends React.Component {
   constructor(props) {
     super(props);
-    this.state = { ...this.props, current: { ...this.props.currentState }, errors: {}, modified: {}, isEditMode: window.location.href.includes('edit') };
+    this.state = { ...this.props, current: { ...this.props.currentState }, errors: {}, modified: {}, isedit_mode: window.location.href.includes('edit') };
```

Comment:
It looks like you're trying to make all of these variables consistently snake case (which is something that should be discussed as a team at some point because we mix snake case and camelcase on the frontend) - I don't think it's super necessary to change this, but this should be `is_edit_mode` if so.

Reply:
good catch - reverted